src/StrawberryShake/Tooling/src/Configuration/VisibilityOverrides.cs

```diff
@@ -0,0 +1,8 @@
+namespace StrawberryShake.Tools.Configuration
+{
+    public class VisibilityOverrides
+    {
```

Comment:
And this would then follow suite and be `AccessModifiers`

Reply:
Done.